lib/diagnostic.rb

```diff
@@ -93,13 +98,15 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.pop(2)
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = '12, 34, nil, nil, 99'
+
+# is this just arr[4] = 99 ?
```

Comment:
That's exactly what it is ^^ 
Correct! but you forgot the brackets around your array